frontend/app/openproject-app.js

```diff
@@ -203,7 +203,8 @@ openprojectApp
       $httpProvider.interceptors.push(function($q) {
         return {
           'request': function(config) {
-            if (!config.url.match('(^/templates|\\.html$)')) {
+            if (!config.url.match('(^/templates|\\.html$)') &&
```

Comment:
@0xF013 thanks for the feedback. I will make the changes you suggested. 

What I was most interested in is whether this was the right place to do it as to my understanding this is employed for every request performed via $http. Are you aware of any unforeseen consequences this might have? (Apart from the edge case of choosing to have his/her instance running under something like `/work_packages` which will probably break)